### PR TITLE
docs(core): update release blog post

### PR DESCRIPTION
**Pull request by mhartington**
Provide a recap of features from the previous release

## Related Issue(s)

Fixes #

### CODE REVIEW

File: docs/blog/2024-10-03-nx-20-release.md [hunk 1]
```diff
@@ -6,7 +6,15 @@ tags: [nx, release]
 cover_image: /blog/images/2024-10-03/nx-20-header.png
 ---
 
-I know it's hard to believe but Nx 20 is here! There's a lot to cover in this release, so let's just dive right in.
+I know it's hard to believe but Nx 20 is here! There's a lot of great updates in this release, but look back at some of the major features from Nx 19:
+
+- Added [`nx import`](nx-import) to bring import existing projects while maintaining git history.
```
Comment (isaacplmann) [inline docs/blog/2024-10-03-nx-20-release.md:11]:
```suggestion
- Added [`nx import`](nx-import) to import existing external projects while maintaining git history.
```

File: docs/blog/2024-10-03-nx-20-release.md [hunk 2]
```diff
@@ -6,7 +6,15 @@ tags: [nx, release]
...
+- Added a [generated](crystalize-command) to convert all your Nx Plugins to use inferred tasks.
```
Comment (isaacplmann) [inline docs/blog/2024-10-03-nx-20-release.md:12]:
```suggestion
- Added a [generator](crystalize-command) to convert all your Nx Plugins to use inferred tasks.
```

File: docs/blog/2024-10-03-nx-20-release.md [hunk 3]
```diff
@@ -6,7 +6,15 @@ tags: [nx, release]
...
+- Moved to [rspack](nx-rspack) as the default bundler for Module Federation
+- Added [file-based versioning](file-based-versioning) for Nx Release
+- A new [plugin for Gradle](nx-gradle) and support for [atomized tasks](gradle-atomizer)
```
Comment (isaacplmann) [inline docs/blog/2024-10-03-nx-20-release.md:15]:
```suggestion
- Created a new [plugin for Gradle](nx-gradle) and support for [atomized tasks](gradle-atomizer)
```

File: docs/blog/2024-10-03-nx-20-release.md [hunk 4]
```diff
@@ -6,7 +6,15 @@ tags: [nx, release]
...
+
+With Nx 20 we're keeping the momentum going and bringing some quality improvments to the overall monorepo experience as well a boost in speed for TypeScript projects.
```
Comment (isaacplmann) [inline docs/blog/2024-10-03-nx-20-release.md:17]:
```suggestion
With Nx 20 we're keeping the momentum going and bringing some quality improvements to the overall monorepo experience as well as a boost in speed for TypeScript projects.
```

Comment (isaacplmann) [inline docs/blog/2024-10-03-nx-20-release.md:17]:
```suggestion
With Nx 20 we're keeping the momentum going and bringing some quality improvements to the overall monorepo experience as well as a boost in speed for TypeScript projects.
```

**Review by isaacplmann — CHANGES_REQUESTED** [review]
Typos

File: docs/blog/2024-10-03-nx-20-release.md (hunk 1 above)
Comment (juristr) [inline docs/blog/2024-10-03-nx-20-release.md:11]: where should all these links point to? like `nx-import`, `crystalize-command` etc? they don't work right now.

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**